salt/top.sls

```diff
@@ -138,3 +138,6 @@ base:
     - match: grain
     - scylladb.configure
     - scylladb.tests
+  'G@roles:reddit and P@environment:(rc|production)-apps':
```

Comment:
This doesn't need to be predicated on environment. We should just match on the `roles:reddit` grain with a match type of grain.